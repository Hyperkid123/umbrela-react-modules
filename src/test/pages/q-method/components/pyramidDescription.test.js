import React from 'react'
import PyramidDescription from '../../../../pages/q-method/components/pyramidDescription';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Q-method navigation component', () => {
  it('should render correclty' , () => {
      const tree = shallow(
        <PyramidDescription/>
      ).dive();
      expect(toJson(tree)).toMatchSnapshot();
  });
});
