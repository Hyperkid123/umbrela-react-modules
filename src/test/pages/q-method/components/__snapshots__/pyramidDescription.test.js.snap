// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Q-method navigation component should render correclty 1`] = `
<div
  className="sc-bdVaJa fizmbb"
>
  <styled.div
    grow={true}
    horizintalCenter={true}
    verticalCenter={true}
  >
    <span>
      Náhled rozdělení stimulů
    </span>
  </styled.div>
  <styled.div>
    <span>
      Přidat nebo odebrat pole
    </span>
  </styled.div>
</div>
`;
